README.md

```diff
@@ -1,2 +1,33 @@
-# awesome-innersource
-Awesome InnerSource (per [sindresorhus/awesome](https://github.com/sindresorhus/awesome/blob/main/awesome.md))
+# Awesome InnerSource [![Awesome](https://awesome.re/badge-flat2.svg)](https://awesome.re)
+
+<!--lint ignore double-link-->
```

Comment:
Do you want to use a GitHub Action to do the linting?

Reply:
Yes, however let's add awesome-linter separately, keeping the PR's small